src/apollo/types.ts

```diff
@@ -1,5 +1,7 @@
+import { FetchResult, NextLink, Operation } from 'apollo-link';
 import { BatchLink } from 'apollo-link-batch';
 import { GraphQLSchema } from 'graphql/type';
+import { Observer } from 'zen-observable-ts';
```

Comment:
although this is included by apollo-link in node_modules,
but should we add this as an explicit dependency in package.json?